src/BleOtaUploader.cpp

```diff
@@ -127,7 +127,11 @@ void BleOtaUploader::handleBegin(const uint8_t* data, size_t length)
     currentLength = 0;
     uploading = true;
     crc.reset();
-
+    crc.setPolynome(0x04C11DB7);
+    crc.setStartXOR(0xFFFFFFFF);
+    crc.setEndXOR(0xFFFFFFFF);
+    crc.setReverseIn(true);
+    crc.setReverseOut(true);
```

Comment:
A lot of combinations, I'll check if there is easier way

Reply:
Created issue: https://github.com/RobTillaart/CRC/issues/31
Lets see what author think about it